openlibrary/plugins/worksearch/autocomplete.py

```diff
@@ -21,7 +21,7 @@ def to_json(d):
 
 class autocomplete(delegate.page):
     path = "/_autocomplete"
-    fq = ['-type:edition']
+    fq: tuple[str, ...] = ('-type:edition',)
```

Comment:
I think the type hints here aren't actually adding anything to the developer experience or protection ; they are making the code look a little more complex than it needs to be. Just converting to a tuple should be enough. But give it a go, cause this might trigger other type errors down stream!

```suggestion
    fq = ('-type:edition',)
```

Ditto in the other spots. @RayBB confirmed, and it looks like this does not cause issues!

Reply:
Yeah, it worked and no issues occurred.